cypress.json

```diff
@@ -3,5 +3,5 @@
   "viewportHeight": 568,
   "viewportWidth": 320,
   "projectId": "8r2xra",
-  "defaultCommandTimeout": 8000
+  "defaultCommandTimeout": 15000
```

Comment:
a test case that includes typing "atchakosuk" is flaky in my local testing. It times out, which is reasonable cuz typing a bunch of stuff in succesion strains the server.

Reply:
yikes :/